lib/topological_inventory/persister/logging.rb

```diff
@@ -0,0 +1,17 @@
+module TopologicalInventory
+  module Persister
+    class << self
+      attr_writer :logger
+    end
+
+    def self.logger
+      @logger ||= Logger.new(STDOUT, :level => Logger::ERROR)
```

Comment:
Eventually we are going to need to do common logging...we should create a work task for that.

Reply:
Just need a gem name.  `top_logic-logging`?